FirebaseAI/Tests/Unit/Types/ToolTests.swift

```diff
@@ -38,18 +40,31 @@ final class ToolTests: XCTestCase {
     """)
   }
 
+  func testEncodeTool_codeExecution() throws {
+    let tool = Tool.codeExecution()
+
+    let jsonData = try encoder.encode(tool)
+
+    let jsonString = try XCTUnwrap(String(data: jsonData, encoding: .utf8))
+    XCTAssertEqual(jsonString, """
+    {
+      "codeExecution" : {
+
+      }
+    }
+    """)
+  }
+
   func testEncodeTool_functionDeclarations() throws {
     let functionDecl = FunctionDeclaration(
       name: "test_function",
       description: "A test function.",
       parameters: ["param1": .string()]
     )
     let tool = Tool.functionDeclarations([functionDecl])
-
-    encoder.outputFormatting.insert(.withoutEscapingSlashes)
```

Comment:
Why this change?

Reply:
I realized this one was already set in the test [`setUp()`](https://github.com/firebase/firebase-ios-sdk/pull/15280/files/b52a408ed7dcdc11882df6124dc2f5a77f3128e0#diff-1bf07f6983c8ea9d3010aa30de47420bde70107e47b4c828e68bb10a728a60e0L25) so it was a no-op. The other whitespace changes were just me being pedantic about using arrange-act-assert structure in tests (Google-internal link [Unit Testing Best Practices](https://goto.corp.google.com/unit-testing-practices#structure)).